Update documentation for `cargo fix --broken-code` (#120)
rust-lang-nursery/edition-guide: src/editions/transitioning-an-existing-project-to-a-new-edition.md

keyword](../rust-2018/trait-system/dyn-trait-for-trait-objects.html) for
trait objects.

> We ideally want `cargo fix` to fix all these idioms automatically in the same

Reviewer: ```suggestion
Eventually, we want `cargo fix` to fix all these idioms automatically in the same
```

> manner we did for upgrading to the 2018 edition. **At this point in time,

Reviewer: ```suggestion
manner we did for upgrading to the 2018 edition. **Currently,
```

> though, the "idiom lints" are not ready for widespread automatic fixing.** The

Reviewer: ```suggestion
though, the *"idiom lints"* are not ready for widespread automatic fixing.** The
```

compiler isn't making `cargo fix`-compatible suggestions in many cases right

> now (and is making incorrect suggestions in others), so enabling the idiom

Reviewer: ```suggestion
now. In other cases it even makes incorrect suggestions. Therefore, enabling the idiom
```
breaking up long sentence.

lints, even with `cargo fix`, is likely to leave your crate with many warnings
still remaining.

> It's planned to make these idiom migrations a seamless part of the Rust 2018

Reviewer: ```suggestion
We have plans to make these idiom migrations a seamless part of the Rust 2018
```

experience, but we're not there yet. As a result the following instructions are

> recommended only for the intrepid who are willing to work through a few

Reviewer: intrepid is a rare word; use something else?

If `cargo fix` can't fix something, it will print a warning to the console,
and you'll have to fix it manually.

> Also as mentioned before, there are known bugs around the idiom lints which

Reviewer: ```suggestion
As mentioned before, there are known bugs around the idiom lints which
```

> means they're not all ready for prime time yet. This means that you may get a

Reviewer: ```suggestion
means they're not all ready for prime time yet. Thus, you may get a
```
Means and then means again; reduce repetition;

scary-looking warning to report a bug to Cargo, which happens whenever a fix

> proposed by rustc actually caused code to stop compiling by accident. If you'd

Reviewer: ```suggestion
proposed by `rustc` actually caused code to stop compiling by accident. If you'd
```

like `cargo fix` to make as much progress as possible, even if it causes code to
stop compiling, you can execute:

```console
$ cargo fix --edition-idioms --broken-code
```

> This will instruct `cargo fix` to apply changes even if they cause code to stop

Reviewer: you already said "stop compiling" so the bit after "even" feels redundant either here or in the previous location.

compiling, and you'll see the compilation result after all fixes are applied. If

> you notice anything awry, please feel free to report an issue to Cargo and we'll

Reviewer: ```suggestion
you notice anything wrong or unusual, please feel free to report an issue to Cargo and we'll
```
